manifests/p/PostgreSQL/pgAdmin/5.4/PostgreSQL.pgAdmin.installer.yaml

```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: PostgreSQL.pgAdmin
+PackageVersion: 5.4
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v5.4/windows/pgadmin4-5.4-x64.exe
+  InstallerSha256: 486CD09D9F8CE596A2B66AEF7A2DC032126551236A6471D720D681997E243D05
+  InstallerSwitches:
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT
+    Custom: /SUPPRESSMSGBOXES /ALLUSERS
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: inno
+  Scope: user
+  InstallerUrl: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v5.4/windows/pgadmin4-5.4-x64.exe
+  InstallerSha256: 486CD09D9F8CE596A2B66AEF7A2DC032126551236A6471D720D681997E243D05
+  InstallerSwitches:
+    Silent: /VERYSILENT
+    SilentWithProgress: /SILENT
+    Custom: /SUPPRESSMSGBOXES /CURRENTUSER
```

Comment:
When the InstallerType is Inno/Nullsoft etc you don't need to add Silent and SilentWithProgress switches btw

Reply:
Thanks, good to know. Those switches are already baked into winget, I assume?
I could clean up the existing manifests, but that's probably not necessary?

Reply:
Yup atleast /VERYSILENT and /SILENT type of switches, imo it should also include /SUPPRESSMSGBOXES and /NORESTART (or detect it and prompt user at the end of installation/upgrade), but for now we just use those when needed (/NORESTART most of the time just because its nice and dosn't break anything)
Naaah just clean it up on the next PR :)

Reply:
Agreed (esp. detect & prompt) and will do.